content/manuals/security/for-admins/roles-and-permissions.md

```diff
@@ -110,12 +110,9 @@ The following table outlines Docker Build Cloud management permissions for membe
 
 | Permission                   | Member | Editor | Organization owner |
 | ---------------------------- | :----- | :----- | :----------------- |
-| Sign up for starter plan     | ✅     | ✅     | ✅                 |
-| Use a cloud builder          | ✅ \*  | ✅ \*  | ✅ \*              |
-| Manage seat allocation       | ✅     | ✅     | ✅                 |
+| Use a cloud builder          | ✅     | ✅     | ✅                 |
 | Create and remove builders   | ✅     | ✅     | ✅                 |
 | Buy seats or reduce seat cap | ❌     | ❌     | ✅                 |
+| Configure builder settings   | ❌     | ❌     | ✅                 |
```

Comment:
@zappy-shu anything else that comes to mind and we should include?

Reply:
Looking at the rules: currently all members can configure builder settings, not just owners